src/components/Questions.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react"
+
+function Question({allAns,correct_ans,isShow,answer, setAnswer,setCounter}){
```

Comment:
You can use Prettier or another code formatter to format your code.
```suggestion
function Question({allAns, correct_ans, isShow, answer, setAnswer, setCounter}){
```